src/editor/map/map_context.cpp

```diff
@@ -595,7 +604,7 @@ bool map_context::save_scenario()
 	try {
 		std::stringstream wml_stream;
 		wml_stream
-			<< "# This file was generated using the scenario editor.\n"
+			<< editor_generated << "\n"
```

Comment:
Why does exactly one line of this move to the .hpp file? Why not all or none?

Reply:
What happened was basically:
0) "It's silly that the editor isn't writing out a valid scenario cfg."
1) I added the [multiplayer] tag.
2) I realized that the presence of a top-level tag was being used by the editor to determine if it was an editor generated scenario or not, which wouldn't work anymore.
3) I decided to instead use the presence of that comment string to determine if the file was an editor generated scenario or not.
4) I decided that was also horrible, and I should just use file extensions instead.

Short version - I should just move it back to the cpp file with the rest of that comment block.